src/libraries/System.IO.FileSystem/tests/AppExecLinks.Windows.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.IO.Enumeration;
+using System.Linq;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    public class AppExecLinks : BaseSymbolicLinks
+    {
+        [ConditionalTheory(typeof(PlatformDetection), nameof(PlatformDetection.HasUsableAppExecLinksDirectory))]
```

Comment:
Sorry if I skipped this - will this throw if HasUsableAppExecLinksDirectory return false or the test could be skipped always silently in CIs?

Reply:
It should return false and cause the test to be skipped due to the absence of a WindowsApps dir.

The property this attribute is pointing to, should not throw. If it throws, it's a bug and needs to be fixed.

Reply:
My concern is that in the case the tests can be never really evaluated on CIs.